src/styles/constants/_media-queries.scss

```
@@ -1,10 +1,12 @@
 $size-small-max: 374px;
+$size-small-medium-max: 460px;
 $size-medium-max: 767px;
 $size-large-max: 1023px;
 
 $small-screen: unquote('(max-width: #{$size-small-max})');
 $medium-screen: unquote('(max-width: #{$size-medium-max})');
 $large-screen: unquote('(max-width: #{$size-large-max})');
+$small-medium-screen: unquote('(max-width: #{$size-small-medium-max})');
```

Looks like these should be sorted by the `px` amount?

```suggestion
$small-screen: unquote('(max-width: #{$size-small-max})');
$small-medium-screen: unquote('(max-width: #{$size-small-medium-max})');
$medium-screen: unquote('(max-width: #{$size-medium-max})');
$large-screen: unquote('(max-width: #{$size-large-max})');
```

---

Same suggestion applies for the new additions in `src/styles/variables.js`, `src/styles/variables.json`, and `src/styles/variables.ts`.

---

Good idea. Done.